src/saving/SaveList.tscn

```diff
@@ -39,4 +40,12 @@ size_flags_vertical = 5
 popup_exclusive = true
 window_title = "Delete this Save?"
 dialog_text = "Text not set..."
+
+[node name="LoadConfirmDialog" type="ConfirmationDialog" parent="VBoxContainer"]
+size_flags_horizontal = 5
+size_flags_vertical = 5
+popup_exclusive = true
+window_title = "Load this save?"
+dialog_text = "This save is from an different version of Thrive and may be incompatible. Do you want to load it anyway?"
```

Comment:
I suggest adding "As Thrive is currently early in development save compatibility is not a huge priority. If you have issues you can report them, but they aren't the highest priority right now." before the "Do you want to load it anyway?" part.
Also it'd be nice to include the version numbers in the popup text so the player can see how different the version numbers are.

One suggestion that you don't have to do but would be nice: if the version number of the save is higher than current, there would be a much bigger warning saying it probably won't work and update your game.

Reply:
Is there a version comparison function already implemented and if not, where should I put it?

Reply:
I don't think there is. Maybe a new file in src/general. If there is a suitably licensed semver library for C# that could be used as well.